README.md

```diff
@@ -101,6 +101,12 @@ typically in your root component, and customize the values of its properties in
   constructor(private config: NgSelectConfig) {
       this.config.notFoundText = 'Custom not found';
       this.config.appendTo = 'body';
+      // set the bindValue to global config when you use the same 
+      // bindValue is most of the place. 
```

Comment:
typo: _bindValue **is** most of the place_

Reply:
Fixed the typo :)